app/lib/application_service.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationService < ApplicationController
-  def self.call
-    new.call
+  def self.call(*params)
+    new.call(*params)
```

Comment:
I would use more generic name here - arguments or `args`:

```suggestion
  def self.call(*args)
    new.call(*args)
```

Reply:
I agree with that.